language-support/ts/codegen/tests/npm_check.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This test checks that the package-lock.json file gets updated by npm when a generated dependency
+# changes.
+
+set -eou pipefail
+
+# --- begin runfiles.bash initialization v2 ---
+# Copy-pasted from the Bazel Bash runfiles library v2.
+    set -uo pipefail; f=bazel_tools/tools/bash/runfiles/runfiles.bash
+    source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
+      source "$(grep -sm1 "^$f " "${RUNFILES_MANIFEST_FILE:-/dev/null}" | cut -f2- -d' ')" 2>/dev/null || \
+      source "$0.runfiles/$f" 2>/dev/null || \
+      source "$(grep -sm1 "^$f " "$0.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
+      source "$(grep -sm1 "^$f " "$0.exe.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
+      { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
```

Comment:
Probably worth removing the duplicated `set`s at the beginning and end and fixing the indentation.

Reply:
this is copy-pasted (you can’t avoid it, this is a bootstrapping problem, this is the logic for locating dependencies, if you try to factor it out, you can’t locate the factored out code). I wouldn’t modify this in any way.

Reply:
I hoped that removing the `set`s and changing the indentation was safe enough, but I trust your judgement. 🙂